thrift/from_wire.go

```diff
@@ -123,6 +123,15 @@ func valueFromWireMap(spec *compile.MapSpec, w wire.Map) (map[string]interface{}
 	return result, nil
 }
 
+func mapEnumValueToName(enumSpec *compile.EnumSpec, result int32) interface{} {
+	for _, item := range enumSpec.Items {
```

Comment:
We can have a reverse mapping but it will have undefined behavior when multiple enum items have the same enum value.